dnd5e/module/item/entity.d.ts

```diff
@@ -555,14 +536,59 @@ declare namespace Item5e {
     }
   }
 
-  interface Data extends Item.Data<Data.Data> {
-    data: Data.Data;
-    effects: ActiveEffect.Data[];
-    img: string;
-    name: string;
-    permission: Entity.Permission;
-    sort: number;
-    type: DND5e.ItemType;
+  namespace Labels {
+    // hasOwnProperty('activation')
```

Comment:
I believe you should remove these comments

Reply:
sounds good